src/components/icons/ChatIcon.tsx

```diff
@@ -1,22 +1,22 @@
 import { SVGAttributes } from "react";
 
-function ChatIcon(props: SVGAttributes<SVGElement>) {
+const ChatIcon = (props: SVGAttributes<SVGElement>) => (
   <svg
-    xmlns='http://www.w3.org/2000/svg'
-    width='24'
-    height='24'
-    fill='none'
-    viewBox='0 0 24 24'
+    xmlns="http://www.w3.org/2000/svg"
+    width="24"
```

Comment:
should we have these hardcoded? shouldn't they come from tokens?

Reply:
we apply a wrapper to the icon where we overwrite these properties, so that we have one single source of truth for all the icons without having to rely on the details in each single icon file